pypowsybl/network/impl/pandapower_converter.py

```diff
@@ -13,9 +13,14 @@
 import numpy as np
 import pandas as pd
 import pypowsybl._pypowsybl as _pp
-from pandapower import pandapowerNet
 from pandas import Series
 from pandas import DataFrame
+try:
+    from pandapower import pandapowerNet
```

Comment:
This is not useful to warm the user until he wants to use it. There is already such a warning in the convert function